src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2260,6 +2260,20 @@ public static int getStreamingConnectionsPerHost()
         return conf.streaming_connections_per_host;
     }
 
+    public static boolean isFullSSTableTransfersEnabled()
+    {
+        if (conf.server_encryption_options.enabled || conf.server_encryption_options.optional)
+        {
+            logger.debug("Internode encryption enabled. Disabling zero copy SSTable transfers for streaming.");
```

Comment:
Nobody will ever see this at `debug` level. We should at minimum `warn` if `streaming_zerocopy_sstables_enabled` is and internode encryption are both enabled at the same time.

Reply:
Done